src/crypto/cmac.js

```diff
@@ -73,20 +72,25 @@ export default async function CMAC(key) {
 }
 
 async function CBC(key) {
-  if (util.getWebCrypto() && key.length !== 24) { // WebCrypto (no 192 bit support) see: https://www.chromium.org/blink/webcrypto#TOC-AES-support
-    key = await webCrypto.importKey('raw', key, { name: 'AES-CBC', length: key.length * 8 }, false, ['encrypt']);
-    return async function(pt) {
-      const ct = await webCrypto.encrypt({ name: 'AES-CBC', iv: zeroBlock, length: blockLength * 8 }, key, pt);
-      return new Uint8Array(ct).subarray(0, ct.byteLength - blockLength);
-    };
-  }
   if (util.getNodeCrypto()) { // Node crypto library
     return async function(pt) {
       const en = new nodeCrypto.createCipheriv('aes-' + (key.length * 8) + '-cbc', key, zeroBlock);
       const ct = en.update(pt);
       return new Uint8Array(ct);
     };
   }
+  try {
+    if (util.getWebCrypto()) {
+      key = await webCrypto.importKey('raw', key, { name: 'AES-CBC', length: key.length * 8 }, false, ['encrypt']);
+      return async function(pt) {
+        const ct = await webCrypto.encrypt({ name: 'AES-CBC', iv: zeroBlock, length: blockLength * 8 }, key, pt);
+        return new Uint8Array(ct).subarray(0, ct.byteLength - blockLength);
+      };
+    }
+  } catch (err) {
+    // no 192 bit support in Chromium, see: https://www.chromium.org/blink/webcrypto#TOC-AES-support
+    if (err.name !== 'OperationError') throw err;
```

Comment:
The problem I have with `NotSupportedError` is that we'd have to potentially add it everywhere, and I'd do that only if there is a relevant use-case. AES-GCM and CBC have widespread support.
It seems that Chrome not implementing one AES size is not spec-respecting to begin with: https://github.com/w3c/webcrypto/issues/62 . 
> As the spec is currently written, it is not an option to support AES-GCM but to not support one of the key sizes. 

I'd like to be as strict as possible with these fallbacks as far as decryption goes, since it's sensitive. In this sense, ;ooking at the other mode's code, I think we should probably add a browser + algo check , when it comes to Chrome's OperationError.

With encryption, it's not as important, but since the fallback is unexpected at this point, I would still write specific code, so that we can keep track of other special cases too (and decide how important they are if we want to drop support).

Reply:
Yeah, it's not compliant, but at the same time the existence of a spec can't force implementations to implement stuff they don't want to. The spec should accommodate that to some extent, and if we need to add a provision for it, allowing to throw `NotSupportedError` would make much more sense than allowing to throw `OperationError` - so we shouldn't hardcode the latter.

But yes, we can add a browser + key length check instead :+1:

Reply:
We do catch [specific browser errors in other parts](https://github.com/openpgpjs/openpgpjs/blob/main/src/crypto/public_key/elliptic/ecdsa.js#L62). I can add `NotSupportedError` as an additional condition, but I think it's important to keep checking for exceptional browser-specific errors too, rather than fallback on all cases (which has given issues in the past with e.g. RSA signature verification in the context of key validation, for instance).
Not all browsers versions give the same error, nor do they fail to begin with, that's why we originally introduced error checking rather than platform checking.

Reply:
Alrighty, yeah, seems good to me :+1:
(Old versions of Chrome did throw `NotSupportedError` instead of `OperationError`, though to me the more important part is keeping open the possibility of someone throwing that in the future)